### PR TITLE
Moving from RHEL to Centos base image

### PR DESCRIPTION
**Pull request by Madhan-SWE**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Moved from RHEL to centos base image.
Fixed issues in centos base driver image.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,9 +23,7 @@ COPY --from=builder /go/src/sigs.k8s.io/ibm-powervs-block-csi-driver/bin/ibm-pow
 ENTRYPOINT ["/ibm-powervs-block-csi-driver"]
 
 
-FROM registry.access.redhat.com/ubi8/ubi:8.5 AS rhel-base
-RUN yum --disableplugin=subscription-manager -y install httpd php \
-  && yum --disableplugin=subscription-manager clean all
-RUN clean-install ca-certificates e2fsprogs mount udev util-linux xfsprogs
-COPY --from=builder /go/src/sigs.k8s.io/ibm-powervs-block-csi-driver /ibm-powervs-block-csi-driver
+FROM quay.io/centos/centos:stream9 AS centos-base
```
Comment (mkumatag) [inline Dockerfile:26]: stream9 is still in beta stage, lets stick to stream8 for now.

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (Madhan-SWE) [reply Dockerfile:26]: There were some security vulnerabilities in stream8.
that's why I moved to stream9 and the vulnerabilities are not seen now.
Do you think we still need to move to stream8.

Comment (Madhan-SWE) [reply Dockerfile]: stream changed to stream8 as discussed

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Comment by Madhan-SWE** [comment]
@mkumatag can you merge this?

**Pull request merged**